x/stakeibc/keeper/icqcallbacks_delegator_shares_test.go

```diff
@@ -375,3 +375,24 @@ func (s *KeeperTestSuite) TestDelegatorSharesCallback_PrecisionError() {
 	expectedValDelegation := tc.hostZone.Validators[tc.valIndexQueried].Delegation.Add(precisionErrorTokens)
 	s.Require().Equal(expectedValDelegation.Int64(), validator.Delegation.Int64(), "validator delegation amount")
 }
+
+func (s *KeeperTestSuite) TestCalibrateDelegationCallback_Successful() {
```

Comment:
Should this also test the case where an update tries to change the delegation by greater than `CalibrationThreshold`?

Reply:
Added a test for this. Required some additional scaffolding; if you have ideas on how to set this test up without that scaffolding please lmk @sampocs!

Reply:
Simplified the setup and added some more coverage [[here](https://github.com/Stride-Labs/stride/pull/1013)]

Reply:
the other PR was merged into here - going to mark this resolved!

Reply:
Looks great, thanks for jumping in on this @sampocs 🙌